lessons/linear-regression/langs/en-us/index.adoc

```diff
@@ -219,31 +191,29 @@ How well can you interpret the results of a linear regression analysis? How woul
 
 [.lesson-instruction]
 - What does it mean when a data point is _above_ the line of best fit?
-** _It means the y-value is _higher_ than the sample would have predicted for that x-value._
+** _It means the y-value is higher than the sample would have predicted for that x-value._
 - What does it mean when a data point is _below_ the line of best fit?
-** _It means the y-value is _lower_ than the sample would have predicted for that x-value._
-- Turn to @printable-exercise{pages/interpreting-regression-lines-n-rvalues.adoc}, and match the write-up on the left with the line of best fit and @math{r}-value on the right.
+** _It means the y-value is lower than the sample would have predicted for that x-value._
+- Turn to @printable-exercise{interpreting-regression-lines-n-rvalues.adoc}, and fill in the blanks for each scenario on the left using information from the @vocab{predictor function} and @vocab{r-value} on the right.
 
 Let's take a look at how the Data Cycle can be used with Linear Regression, and how the result can be used to form our Data Story.
 
 [.lesson-instruction]
-Read @printable-exercise{pages/regression-analysis-example.adoc}.
-
-Have students explain the connection between the Ask Questions and Consider Data step. Do they match? Why or why not?
-
-At the bottom of the page we have the _Data Story_ for this question, which includes the results of the analysis and a responsible way to write about them. When looking at a regression for adoption time v. age for just the cats, we saw that the slope of the predictor function was +0.23, meaning that for **every year older, we expect a cat to take +0.23-weeks longer to be adopted**. The @math{r}-value was +0.566, confirming that the correlation is positive and indicating moderate strength.
-
-
+- Read @printable-exercise{pages/regression-analysis-example.adoc}.
+- What do you Notice? What do you Wonder?
+- Have students explain the connection between the Ask Questions and Consider Data step. Do they match? Why or why not?
```

Comment:
The first question is written as if it's spoken directly to students, but the second is written as if it's spoken to teachers, and the sample answer is written as if it comes from students. Can we make the wording of the second question student-facing, so we're consistent?